layouts/posts/single.html

```diff
@@ -33,7 +33,7 @@ <h1 class="title">
 
       <div class="post-content">
         {{ if .Params.featuredImage }}
-          <img src="{{ .Params.featuredImage | relURL }}" alt="Featured image"/>
+          <img src="{{ or (.Resources.GetMatch .Params.featuredImage) (resources.Get .Params.featuredImage).RelPermalink (.Params.featuredImage | relURL) }}" alt="Featured image"/>
```

Comment:
Hmm... I don't know. This looks like overly complicated. `GetMatch` could be a problem, because of its pattern matching it could match with a similarly named file. And, `resources.Get` will try to find the image inside the `assets` folder, which is not related.

I see two possible solutions for your problem.
1. Just removing `relURL` from the original line, so you could use relative path to the page and not to the base URL. (It makes sense, but we'll need to remove `relUr` from the favicon and avatar code too.)
2. Use a full path in `featureImage`  such as `posts/page-bundle/images/something.jpg`.

I don't mind going with the option (1), but people will complain about the breaking change.

Reply:
I suppose I could use a full path in the featured image, it would just be nice to be able to use a relative path like you can with page resources. I definitely don't want to introduce a breaking change.

I don't know that it's too complicated. The goal is to enable the option for bundled images with a relative path, and whatever evaluates truthy first is picked.

That said, I've done some more testing and realized that if I define a page resource with the same path, the image doesn't render correctly, so I suppose I can see the issue there.

I think I will just use an absolute path, or override the `single.html` template with my own.

Reply:
Yeah, currently this is a practical option. But let me better check other solution this weekend. I think we could create a custom template to render image links and abstract this code. I'm also thinking that using the asset pipeline for images could be a good idea, but I need to understand it better first.

Reply:
Sure thing. I'm relatively new to using the asset pipeline as well. If you'd like I can draft something in this PR or we can close it and I'll let you do your thing.